pennylane_lightning/core/lightning_base.py

```diff
@@ -55,7 +55,7 @@ class LightningBase(QubitDevice):
             OpenMP.
     """
 
-    pennylane_requires = ">=0.36"
+    pennylane_requires = ">=0.37"
```

Comment:
Shouldn't this requirement be in `pyproject.toml` and `setup.py`? :thinking: 

How do we use this `pennylane_requires` in the code base? Is there really a runtime check for the version of PL?

Reply:
I think this is an old API plugin attribute. It isn't really in use, just informational.

Reply:
I confirm that this is old API stuff. We can probably remove that. You better check with Core, though.